### PR TITLE
Jb54474

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the very last commit belongs to this PR

**Comment by martyone** [comment]
macOS and Windows behavior is to be assessed yet.

File: src/tools/sfdk/remoteprocess.cpp [hunk 1]
```diff
@@ -28,6 +28,7 @@
 #include "textutils.h"
```
Comment (vigejolla) [inline src/tools/sfdk/remoteprocess.cpp:28]: Please update the copyright year

**Comment by vigejolla** [comment]
so far looks good

File: src/tools/sfdk/remoteprocess.cpp (hunk 1 above)
Comment (martyone) [reply src/tools/sfdk/remoteprocess.cpp]: done

**Comment by martyone** [comment]
Debugged on mac and win, polished.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**